finance/management/commands/xformnetfilerawdata.py

```diff
@@ -273,7 +273,7 @@ def load_f460A_row(row, agency, verbosity=1):  # noqa
     beneficiary.save()
 
     # Now we have all the parts, save!
-    money = models.IndependentMoney(  # or calculated_Amount
+    money, _ = models.IndependentMoney.objects.get_or_create(  # or calculated_Amount
```

Comment:
Mostly defensive programming. Especially when it comes to other people's data, best to assume the least possible.